docs/public/_redirects

```diff
@@ -530,6 +530,8 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 /:lang/x/* https://material-ui-x.netlify.app/:lang/x/:splat 200
 
 ## Toolpad
+/toolpad/core/templates/nextjs-dashboard/_next/* https://toolpad-core-nextjs-themed.vercel.app/_next/:splat 200
+/toolpad/core/templates/nextjs-dashboard/* https://toolpad-core-nextjs-themed.vercel.app/:splat 200
```

Comment:
Point 5. added to https://github.com/mui/toolpad/issues/4543, this is currently not hosted under the MUI vercel workspace.